### PR TITLE
FR : Add a few channels

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Answering some channel requests.

Answers #1939 (5 and 7)

Most ones come from an unprotected ISP stream.

### CODE REVIEW

File: channels/fr.m3u [hunk 1]
```diff
@@ -21,8 +23,16 @@ https://bfmlyon-lh.akamaihd.net/i/BFMLYON_KBS@797041/master.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name=" tvg-language="French" tvg-logo="https://i.imgur.com/mihujKt.jpg" group-title="News",BFMTV
+https://bcovlive-a.akamaihd.net/fb3647b53c1a4c278eaecf83230ee4c5/eu-central-1/876450610001/profile_0/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://i.imgur.com/YFEqBS7.png" group-title="",BipTV
 https://biptv.tv/live/biptvstream_orig/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://static.epg.best/es/Boing.es.png" group-title="Kids",BOING
+http://cdn.webtv4.cdnfr.orange.fr/hs/HALO5/hls/live/Boing_live/01.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://static.epg.best/br/Boomerang.br.png" group-title="Kids",Boomerang
+http://cdn.webtv4.cdnfr.orange.fr/hs/HALO5/hls/live/Boomerang_live/01.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://static.epg.best/fr/Cherie25.fr.png" group-title="",Chérie 25
+https://cherie25hls-lh.akamaihd.net/i/cherie25hls_1@579096/master.m3u8
```
Comment (freearhey) [inline channels/fr.m3u:35]: The link doesn't work.
Captura de pantalla 2021-01-25 a las 17 22 46

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/fr.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/fr.m3u:35]: For both ones ?

Comment (freearhey) [reply channels/fr.m3u:35]: No, only: https://cherie25hls-lh.akamaihd.net/i/cherie25hls_1@579096/master.m3u8

Comment (LaneSh4d0w) [reply channels/fr.m3u:35]: Maybe it's geo-locked. I'll check when I can on streamtest.

Comment (LaneSh4d0w) [reply channels/fr.m3u]: Found out it was geolocked. Deleted.

**Review by freearhey — APPROVED** [review]

**Pull request merged**